cypress/support/fragments/orders/orders.js

```diff
@@ -131,5 +131,28 @@ export default {
 
   closeThirdPane: () => {
     cy.do(PaneHeader({ id: 'paneHeaderorder-details' }).find(Button({ icon: 'times' })).click());
-  }
+  },
+
+  getSearchParamsMap(orderNumber, currentDate) {
```

Comment:
there is a list of parameters which should be checked:
<li>Keyword</li>
<li>Date created</li>
<li>Date opened</li>
<li>PO number</li>
Look's like "Date opened" should be added, but see you've already have such check into another method. Add comment about absence of Date opened.

Reply:
Updated